brave/brave-core
# [brave-extension] add settings option to hide Shields activity count

fix https://github.com/brave/brave-browser/issues/3121

## Test Plan:

1. Visit a website with ads/trackers
2. Assert resources blocked count is visible
3.  In a new tab, go to brave://settings/shields/
4. Set the option to show activity count to false
5. Go back to the website
6. Assert resources blocked count is not visible. Refresh page
7. Assert count is still not visible
8. Go back to brave://settings/shields/ and toggle the option `on`
9. Repeat the same operation. Badge count should be visible again
10. Go back to brave://settings/shields/ and toggle the option `off` again
11. Activate the website tab
12. Quit Brave (cancel `npm start` script)
13. Open Brave (`npm start`)
14. Ensure counter does not show

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
